ghcide/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -628,7 +628,14 @@ suggestExtendImport exportsMap contents Diagnostic{_range=_range,..}
           | Just match <- Map.lookup binding (getExportsMap exportsMap)
           , [(ident, _)] <- filter (\(_,m) -> mod == m) (Set.toList match)
            = Just ident
-          | otherwise = Nothing
+
+            -- fallback to using GHC suggestion even though it is not always correct
```

Comment:
I didn't see a test for this branch added - would be good to see it reflected there.

Reply:
Well, I duplicated the test suite to run without `checkParents`, but I just checked and it doesn't cover this branch. It might  be very tricky to construct a test that covers it though, so I'm inclined to leave it until a user complains